.github/actions/setup/action.yml

```diff
@@ -5,7 +5,7 @@ runs:
   steps:
     - uses: actions/setup-node@v3
```

Comment:
Perhaps we should also try moving `setup-node` to v4 as well.  According to setup-node's [release notes](https://github.com/actions/setup-node/releases), v4 changes the action itself to use node 20.

Reply:
Ha! Good prompt, there's a PR for that but it still uses 16.x in the setup file:
https://github.com/OpenZeppelin/openzeppelin-contracts/pull/4702

I'll update here and the other PR should be closed automatically